app/components/App.jsx

```diff
@@ -23,19 +12,7 @@ export default class App extends React.Component {
 
   render() {
     return (
-      <MuiThemeProvider>
```

Comment:
This is a nitpick but MuiThemeProvider is technically not navigation related so should remain in this file and wrap the NavigationComponent.

Reply:
Could you update it to be wrapped in a NavigationComponent? Do now quite know what is the nitpick.

Reply:
update App.jsx to have the MuiTheme since not related to navigation so ...
App.jsx should include:
<Mui...>
<NavigationComponent />
</Mui>